Firestore/tests/Snippet/FirestoreClientTest.php

```diff
@@ -195,7 +195,7 @@ public function testCollectionGroup()
         $snippet->addLocal('firestore', $this->client);
 
         $res = $snippet->invoke();
-        $this->assertEquals('Found 2 documents!', $res->output());
+        $this->assertEquals('2 documents found!', $res->output());
```

Comment:
This change is needed because we dont interpret the `%` in snippets correctly, [ref](https://github.com/googleapis/google-cloud-php/blob/2af4e33d9dddb5f200eddd8827ea62a4b4162acc/Core/src/Testing/Reflection/DescriptionFactory.php#L99-L108)